nequip/utils/global_state.py

```diff
@@ -77,6 +78,9 @@ def set_global_state(
         # === set global seed ===
         seed_everything(123, workers=True, verbose=False)
 
+        # === register default omegaconf resolvers ===
+        _register_resolvers()
```

Comment:
Is there a particular reason why this is done here instead of in `train.py`?

Reply:
So that random helper scripts that set the global state and parse config files can benefit from the resolvers too (there are some that behave this way in the `misc` directory).